src/Data/Pagination/Common/CreatePagination.cs

```diff
@@ -0,0 +1,32 @@
+namespace Data.Pagination.Common
```

Comment:
Should we move this class under the `Data.Pagination` folder? Should we also make sure to adjust the namespaces?

Reply:
done